README.md

```diff
@@ -123,7 +133,11 @@ The configuration for queues/topics can be changed or reset to default on **Demo
 | Request queue routing key  | `inventory.queue.request`  |
 | Response queue routing key | `inventory.queue.response` |
 
-This configuration can be changed in **application.properties** file.
+Part of the configuration can be changed in **application.properties** file.
+
+> This application supports [Direct Reply-to](https://www.rabbitmq.com/direct-reply-to.html) feature, follow the steps to use it:
```

Comment:
I think in our case we just need to specify how could user (someone like Wilhelm) use this default queue in SOAVirt instead of giving general steps to use it.